<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLToolsWidget</class>
 <widget class="QWidget" name="WLToolsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pbUpdateTool">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pbAcceptTool">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbSetHToZero">
     <property name="text">
      <string>all H=0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTableWidget" name="tableTools"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
